spinalcordtoolbox/reports/qc.py

```diff
@@ -152,7 +152,7 @@ def qcslice_layout(x): return x.mosaic()[:2]
         subject=subject,
         plane=plane,
         dpi=dpi,
-        qcslice=qcslice_type,
+        qcslice_type=qcslice_type,
```

Comment:
I agree. My only reservation is that there's a naming conflict, since `qcslice` is already a module-level variable, referring to the `spinalcordtoolbox.reports.slice` module. But maybe I can just rename the imported module to be simply `slice`.